fmlloader/src/main/java/net/minecraftforge/fml/loading/moddiscovery/AbstractJarFileModProvider.java

```diff
@@ -43,15 +32,4 @@ public void scanFile(final IModFile file, final Consumer<Path> pathConsumer) {
         }
         LOGGER.debug(LogMarkers.SCAN,"Scan finished: {}", file);
     }
-
-    @Override
-    public List<IModFile> scanMods() {
```

Comment:
The removal of this method causes a semantics change for anyone calling this function manually, as it will no longer scan mods, instead it will return an empty list, as it is not overridden anywhere else, the new overload is. Can we not fall back on this impl, or forward to `scanMods(Iterable)`?

Does this function actually get called anymore? What happens if someone has a custom mod locator? (ServerPackLocator is probably an example)